addons/hr_holidays/static/src/discuss/core/common/channel_member_list_patch.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="discuss.channel_member" t-inherit-mode="extension">
+        <xpath expr="//*[@t-ref='displayName']" position="inside">
+            <span t-if="member.persona.OutOfOfficeDateEndText" class="text-warning small ms-2 text-truncate ">
+                <i class="fa fa-plane" role="img"/> <t t-esc="member.persona.OutOfOfficeDateEndText"/>
+            </span>
+        </xpath>
```

Comment:
Would remove plane too: there's the IM status that shows the plane, 2 planes next to each other is redundant:
<img width="254" alt="Screenshot 2025-03-27 at 11 23 43" src="https://github.com/user-attachments/assets/7f53d90d-cd68-41d2-825c-eec4596e6c5e" />

_(and would `fw-bold` too, so it's more visible. Usually things under name should have reduced weight than user name, but since this is important makes sense to exceptionally have bold)_

Reply:
If we show "back in X days" everywhere the im status is displayed we might as well get rid of this plane that doesn't work will as a badge imo.

Reply:
Bold seems overkill. Maybe italic to put some emphasis, but the name should remain the most visible part

Reply:
> Bold seems overkill. Maybe italic to put some emphasis, but the name should remain the most visible part

`.fw-bold` is subtle, it's as bold as usual button text, not to confuse with `.fw-bolder` which I agree is too much.
`.fst-italic` would be nice, but Firefox has many bugs of cutting the end of italic text, therefore this is highly discouraged to use italics to stylize text.